lib/ddtrace/contrib/rails/framework.rb

```diff
@@ -18,6 +18,7 @@ module Framework
         DEFAULT_CONFIG = {
           enabled: true,
           auto_instrument: true,
+          auto_instrument_redis: true,
```

Comment:
if we add also ElasticSearch, we should remember to add here the ``auto_instrument_es`` key